umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -72,5 +73,11 @@
         update-format (or update-format (:format concept))
         umm (spec-core/parse-metadata
              context concept-type format metadata {:sanitize? (= :umm-json (mt/format-key update-format))})
-        umm (apply-update update-type umm update-field update-value find-value)]
+        metadata-dates (:MetadataDates umm)
+        metadata-dates-without-update (remove #(= "UPDATE" (:Type %)) metadata-dates)
```

Comment:
I think it would be cleaner and more re-usable if we had a function in `cmr.umm-spec.date-util` to update the update date and then we could call that. Also, we can remove this commented code.